src/ray/common/grpc_util.h

```diff
@@ -11,6 +11,11 @@
 
 namespace ray {
 
+// We already spill large arguments to plasma, however on occasion a task spec will
+// still exceed the default 4MB limit due to a large number of small args. Set the
+// limit larger to avoid crashing in these cases.
+const int kMaxGrpcMessageSize = 100 * 1024 * 1024;
```

Comment:
It might be better to put this in the RayConfig so that users can change it if it's a problem without changing versions

Reply:
+1